btreeidx.cc

```diff
@@ -676,30 +656,6 @@ char const * BtreeIndex::findChainOffsetExactOrPrefix( wstring const & target,
         }
       }
 
-      #if 0
-      DPRINTF( "The winner is %s, compareResult = %d\n", closestString, compareResult );
-
-      if ( closestString != ptr )
-      {
-        char const * left = closestString -1;
-
-        while( left != ptr && left[ -1 ] )
-          --left;
-
-        DPRINTF( "To the left: %s\n", left );
-      }
-      else
-        DPRINTF( "To the lest -- nothing\n" );
-
-      char const * right = closestString + strlen( closestString ) + 1;
-
-      if ( right != leafEnd )
-      {
-        DPRINTF( "To the right: %s\n", right );
-      }
-      else
-        DPRINTF( "To the right -- nothing\n" );
-      #endif
```

Comment:
While I can see the logic of removing obsolete old-Qt code paths, I don't think permanently disabled code paths should be removed. Certainly not in this PR. Not only such changes add more diff to review, removing them is not uncontroversial and could prevent merging this PR, because such code could be useful during development/debugging.

Reply:
agree